### DISCUSSION TITLE
Failed verification - online remote degree

### DISCUSSION THREAD

**Discussion by StephenPearsonDev** [community/community#68233] [GitHub Education]
### Hi everyone,

I have been trying to verify for most of the day, and seem to be just wasting my time. I have been in touch with support through manual tickets but other than the boiler plate responses reitterating the default message from the verify page, I have failed to find resolution. The most recent suggestion is to use github free or pay for pro which beats the purpose of me using an educational account.

My tickets get closed automatically and I have to reopen them, and the previous ones archived. The issue seems to be that I am enrolled on an online degree at a UK university, I am a remote student which means I am not in the country that my university is and I do not have access to any kind of system that I can log into to make it look like I am at their location.

I have provided a university email address and tried to fill out my profile with different details that match my uni details. I have a digital student ID card with my name, date of expiry, student number and various other details. 

So far I have had no luck, the usual replies are that my details need to match my uni details, that I shouldn't use a VPN (well why does github keep telling me I need to be in the uni location?) and that I should provide copies of my university documents - it's 2023 all of my documents are digital and in email form. 

The only option I have for providing verification data is to use my webcam camera, or mobile phone front camera (switching to the back camera crashes the app).

Any support or help with this issue would be appreciate. Or, if possible it would be great if a moderator can confirm that I am inegible for campy by being outside of the country where my uni is located. At least that way I won't need to waste any more of your support teams time by opening tickets.

Thanks

**Comment by milindaShehan** [comment]
I understand the challenges you're facing with the GitHub verification process. It sounds like you've been putting in a lot of effort to resolve this.

Your situation is unique, being an online student at a UK university while residing outside the country. This can definitely complicate the verification process.

From your description, it seems you've tried various approaches, including providing your university email, filling out your profile with matching details, and even attempting to use your digital student ID card. It's disappointing that none of these methods have worked so far.

I completely understand your concern about the GitHub suggestions conflicting with your situation. Additionally, requesting physical copies of documents in this digital age does seem impractical.

You mentioned the option of using your webcam or mobile phone camera for verification. If you haven't already, it might be worth a try to see if this method provides any progress.

I would recommend keeping an eye on your support tickets and ensuring they don't get closed prematurely. It's crucial that your concerns are thoroughly addressed.

I'm hopeful that the GitHub support team will find a suitable resolution for you soon. For more information on GitHub's education verification process, you can visit [this link](https://support.github.com/request/education).

**Reply by StephenPearsonDev** [reply]
Hi,

Thanks for the reply. I have tried using the webcam and the phone to take pictures, that's why I wrote they hadn't worked. As for the premature closing for tickets, the person replying to my tickets closes them as they reply - so it's unlikely I'm missing them, most of my replies have been within a few minutes.

Does this mean I won't be able to verify? 

Edit:

Also to add - in the docs it states that if my documents don't work out I can provide additional evidence via email, is there a way I can get that started? ![image](https://github.com/community/community/assets/96006343/ca89187b-e7a6-46d0-9b95-b4b0c48782cb)

Thanks
